src/app/common/feature-vtabs.component.ts

```diff
@@ -0,0 +1,148 @@
+import { ComponentReference, GLOBAL_MODULES, CertificatesModuleName, WebServerModuleName, AppPoolsModuleName, WebSitesModuleName, FileSystemModuleName, WebServerModuleIcon } from "main/settings";
+import { Component, Input, OnInit, AfterViewInit, forwardRef, ViewChild } from "@angular/core";
+import { OptionsService } from "main/options.service";
+import { UrlUtil } from "utils/url";
+import { LoggerFactory, Logger, LogLevel } from "diagnostics/logger";
+import { VTabsComponent } from "./vtabs.component";
+import { SectionHelper } from "./section.helper";
+
+export const HomeCategory = "Home";
+export class RouteReference {
+    constructor(public name: string, public ico: string, public routerLink: string[]) {}
+}
+
+export const CONTEXT_MODULES = [
+    new RouteReference(WebServerModuleName, WebServerModuleIcon, [`/webserver/${SectionHelper.normalize(WebServerModuleName)}+general`]),
+    new RouteReference(WebSitesModuleName, "fa fa-globe", [`/webserver/${SectionHelper.normalize(WebSitesModuleName)}`]),
+    new RouteReference(AppPoolsModuleName, "fa fa-cogs", [`/webserver/${SectionHelper.normalize(AppPoolsModuleName)}`]),
+    new RouteReference(FileSystemModuleName, "fa fa-files-o", [`/webserver/${SectionHelper.normalize(FileSystemModuleName)}`]),
+];
+
+export class Feature extends ComponentReference {
+    data: any;
```

Comment:
Please avoid using any if we can define type. Or, please explain what is supposed to be set with the data.